File: test-automation-core/src/main/java/com/nortal/test/core/configuration/StackProperties.java
package com.nortal.test.core.configuration;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.stereotype.Component;
import lombok.Data;

@Component
@Data
@EnableConfigurationProperties
@ConfigurationProperties(prefix = "stack")
public class StackProperties {

	private String name;
}

File: test-automation-core-old/src/main/java/com/nortal/test/core/configuration/StackProperties.kt
package com.nortal.test.core.configuration

import lombok.Data
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.stereotype.Component

@Component
@Data
@EnableConfigurationProperties
@ConfigurationProperties(prefix = "stack")
class StackProperties {
    private val name: String? = null
}